### ISSUE TITLE
Font size and color

### ISSUE THREAD

**Issue by motib** [aseba-community/aseba#251]
I think it is important to allow changing the font size to make Aseba accessible to people with vision impairments.

The same is true for syntax coloring since there are people with color-impaired vision. Or, just have an option not to use syntax coloring.

The only thing that I would classify almost as a "bug" and not just a "wish" is the graying-out of comments. Not only does that make them very difficult to read, but it sends the message that comments are not important. Please assign them a "real" color.

Thanks
Moti

**Comment by stephanemagnenat** [comment]
Now that comments are in green, is this issue a duplicate of #376?

**Comment by motib** [comment]
No, it is not a duplicate. I was also referring to the font size. I can still read small fonts, but I find it uncomfortable. I don't know why I opened two separate issues, but please leave open 376 on the font size. Thanks.

**Comment by stephanemagnenat** [comment]
I was thinking of closing this one, not #376

**Comment by motib** [comment]
OK!
